tools/platforms/configs/rbe_default/cc/BUILD

```diff
@@ -51,6 +51,7 @@ cc_toolchain_suite(
         "k8": ":cc-compiler-k8",
         "armeabi-v7a|compiler": ":cc-compiler-armeabi-v7a",
         "armeabi-v7a": ":cc-compiler-armeabi-v7a",
+        "darwin_arm64": ":cc-compiler-darwin-arm64",
```

Comment:
even though the rbe platform is intended for linux, trying to `bazel build //...` hits this and will fail on macOS + arm with an error like below:

```
 bazel build //...
ERROR: /Users/eamorde/Development/kythe/tools/platforms/configs/rbe_default/cc/BUILD:47:19: in cc_toolchain_suite rule //tools/platforms/configs/rbe_default/cc:toolchain: cc_toolchain_suite '//tools/platforms/configs/rbe_default/cc:toolchain' does not contain a toolchain for cpu 'darwin_arm64'
ERROR: /Users/eamorde/Development/kythe/tools/platforms/configs/rbe_default/cc/BUILD:47:19: Analysis of target '//tools/platforms/configs/rbe_default/cc:toolchain' failed
```

Reply:
This is automatically generated and should not be manually updated any changes to it will be overwritten.

Reply:
Ah ok thanks. How is this generated?

Reply:
Found it, looks like it's generated from a tool provided by bazel-toolchains.

Their README as a [section](https://github.com/bazelbuild/bazel-toolchains#custom-execution-properties) on abstracting the RBE platform, perhaps that would solve the issue I was having here. will poke around some more but let me know if you have any suggestions for other ways to fix this

Reply:
Hm I don't see a way to address this here. I'll remove this change for now.

macOS users will need to run:

```
bazel build -- //... -//tools/platforms/configs/rbe_default/...
```

instead of `bazel build //...`